ert_shared/storage/command.py

```diff
@@ -17,8 +17,7 @@ def add_parser_options(ap):
         default=False,
         help="Don't create storage_server.json",
     )
-    ap.add_argument("--blob-url", type=str, default=f"sqlite:///{os.getcwd()}/blobs.db")
     ap.add_argument(
-        "--rdb-url", type=str, default=f"sqlite:///{os.getcwd()}/entities.db"
+        "--rdb-url", type=str, default=f"sqlite:///{os.getcwd()}/ert_storage.db"
```

Comment:
should this be not be `--db-url`?

Reply:
or are we keeping the `relational `

Reply:
i think this goes along with my question and @dotfloat 's answer to that was `Renaming can wait, but alembic can't. We should fix it.`

we could discuss if we should also make the naming correct now - or make a new iteration afterwords 👍

Reply:
I didn't like the `--rdb-url` flag at all and I don't think we should have one. I was also uncertain what the name of such a flag should be.

Unlike the naming of the `ert_storage.db`, this flag can be renamed at any point before the API is stable, which is some time from now.